indigo_app/views/base.py

```diff
@@ -105,3 +105,12 @@ def has_country_permission(self):
                             "but self.country is None.")
 
         return self.request.user.editor.has_country_permission(self.country)
+
+    def doctypes(self):
+        doctypes = settings.INDIGO['DOCTYPES']
+        extras = settings.INDIGO['EXTRA_DOCTYPES'].get(self.country.code)
```

Comment:
I think you'll have to set the default for this to be an empty dict. Otherwise `settings.INDIGO['EXTRA_DOCTYPES']` raises an exception.